src/http/nodegen/request-worker/process.ts

```diff
@@ -1,7 +1,7 @@
 import { Worker } from 'worker_threads';
 import config from '@/config';
 import { WorkerData, WorkerMessage } from './types';
-import { v4 as uuidv4 } from 'uuid';
+import { randomUUID } from 'crypto';
```

Comment:
I think Wish added this as uuidv4 to percent collisions. Which strategy does randomUUID use?

Reply:
v4 in the crypto lib - IIRC older node versions didn't have uuid out of the box, but since we're 18+ now it saves a dep.

i don't think any prod deps use uuid but ill double check